Makefile

```diff
@@ -967,11 +967,14 @@ src/whisper.o: \
 
 $(LIB_WHISPER): \
 	$(OBJ_WHISPER) \
+	$(OBJ_GGML) \
```

Comment:
I don't think it is necessary to include the `ggml` object files in the `whisper` dynamic lib. We want to link `ggml` dynamically

Reply:
ya, removed.